pkg/core/database/postgres/executor.go

```diff
@@ -39,13 +41,19 @@ func (e *Executor) Validate(ctx context.Context) (bool, error) {
 	var dbIdErr error
 	e.dbId, dbIdErr = GetDatabaseId(ctx, pool)
 
+	e.info, _ = GetDatabaseInfo(ctx, pool)
```

Comment:
[non-blocking] If we don't use the error anywhere I'd say remove it altogether. We could also return a default info model so we'd know it errored out